resources/build.sh

```diff
@@ -296,27 +298,23 @@ echo ">> Configure boot FS"
 
 # download base firmware
 mkdir -p ${BOOTFS_PATH}
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/bootcode.bin
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup_cd.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup_db.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup_x.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup4.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup4cd.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup4db.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/fixup4x.dat
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start.elf
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start_cd.elf
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start_db.elf
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start_x.elf
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start4.elf
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start4cd.elf
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start4db.elf
-wget -P ${BOOTFS_PATH} https://github.com/raspberrypi/firmware/raw/master/boot/start4x.elf
+git clone \
+    --bare --depth 1 \
+    --filter=blob:none --no-checkout \
+    https://github.com/raspberrypi/firmware ${BOOTFS_PATH}/.git
+
+(cd ${BOOTFS_PATH} \
+    && git fetch origin ${RPI_FIRMWARE_BRANCH} \
+    && git branch ${RPI_FIRMWARE_BRANCH} FETCH_HEAD \
+    && git --work-tree=${BOOTFS_PATH} checkout ${RPI_FIRMWARE_BRANCH} \
+    -- `git ls-tree refs/heads/${RPI_FIRMWARE_BRANCH} -r --name-only | grep -E "^boot\/([^\/]+)(\.dat|\.elf)$"`)
+
+mv ${BOOTFS_PATH}/boot/* ${BOOTFS_PATH}
+rm -rf ${BOOTFS_PATH}/boot ${BOOTFS_PATH}/.git
 
 # copy linux device trees and overlays to boot
-cp ${ROOTFS_PATH}/boot/dtbs-rpi/*.dtb ${BOOTFS_PATH}/
-cp -r ${ROOTFS_PATH}/boot/dtbs-rpi/overlays ${BOOTFS_PATH}/
+cp ${ROOTFS_PATH}/boot/*.dtb ${BOOTFS_PATH}/
+cp -r ${ROOTFS_PATH}/boot/overlays ${BOOTFS_PATH}/
```

Comment:
This is not working for me:
```
cp: can't stat '/work/root_fs/boot/*.dtb': No such file or directory
```

The directory looks like this:
![image](https://user-images.githubusercontent.com/9787279/105091095-a0435800-5a9f-11eb-8ef9-edc4bf748a3f.png)

Reply:
Oh, thanks! Somehow this worked for me in the actions. Might have been a missing `-e` flag but will investigate this.

Reply:
Here is issue that brought up the change to the paths for the *.dtb's https://github.com/robin-rpr/raspi-alpine-builder/runs/1744298905?check_suite_focus=true#step:6:208

Reply:
Is this maybe different based on the version?
I tested with v3.11 and it seams you are using edge.

That would be bad if there is such a change from version to version.

Reply:
Confirming this issue. Seems like the old folder structure will be deprecated by current `edge`. I would suggest modifying the `cp` to add backwards compatibility.

Reply:
a way to handle both variations would be perfect

Reply:
Give me until the end of next week to fix this up. We are currently re-prioritizing due to some open issues with Rpi builds and plan to support x86_64 first. However we try to keep Arm builds in our backlog and continue working on them later on =)